config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -11,6 +11,7 @@ en:
             THIS ACTION IS IRREVERSIBLE
             
             Are you sure you want to submit? You will no longer be able to amend your submission!
+          confirm_unsubmit: 'Are you sure you want to unsubmit?'
```

Comment:
I'm not sure about v1, but unsubmit seems quite easy to undo here since we can just finalize for the students.